src/main/scala/com/snowplowanalytics/refererparser/Medium.scala

```diff
@@ -13,29 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-package com.snowplowanalytics.refererparser;
+package com.snowplowanalytics.refererparser
 
 /**
- * Enum for the types of referer
- * that we can detect - "medium"
- * in Google's language.
- *
- * @author Alex Dean (@alexatkeplar) <support at snowplowanalytics com>
+ * Enumeration for supported mediums.
  */
-public enum Medium {
-    UNKNOWN,
-    INTERNAL,
-    SEARCH,
-    SOCIAL,
-    EMAIL;
+object Medium extends Enumeration {
```

Comment:
you can have a sealed hierarchy here

Reply:
this wasn't acted on

Reply:
Working on it!

Reply:
ah sorry I thought you were done with my previous feedback